middlewares/habitValidations.js

```diff
@@ -151,7 +156,17 @@ export default {
     }
     next();
   },
+  ensureValidValueOfHabitActive(req, res, next) {
+    const habitActiveBooleanValue =
+      req.body.habitActive && toBooleanValue(req.body.habitActive);
 
+    if (req.body.habitActive && habitActiveBooleanValue === null) {
+      const error = new Error('the habitActive value supplied is not a boolean');
```

Comment:
Could you make the error message here to be "Active status should be 'true' or 'false'" ?